route/build/src/var_lookup.f90

```diff
@@ -234,6 +234,9 @@ MODULE var_lookup
   integer(i4b)     :: KWinflow          = integerMissing  ! 26. KW inflow from upstreams into reach/lake (m3/s)
   integer(i4b)     :: MCinflow          = integerMissing  ! 27. MC inflow from upstreams into reach/lake (m3/s)
   integer(i4b)     :: DWinflow          = integerMissing  ! 28. DW inflow from upstreams into reach/lake (m3/s)
+  integer(i4b)     :: localCC           = integerMissing  ! 29. concentration from local basin (g/m3)
+  integer(i4b)     :: DWsoluteFlux      = integerMissing  ! 30. DW routed solute flux from upstreams into reach/lake (mg/s)
```

Comment:
Does this mean solute can only use the DW method? And can the main flow be something besides DW if the solute is DW?

Reply:
Currently, when DW is used with tracer on, solute mass flux [mg/s] from a reach (`DWsoluteFlux`) and mass [mg] in a reach (`DWsoluteMass`) can be output.  if the other routing methods are on and tracer is on, solutes mass and flux are computed for each routing method, but not be able to output.

Reply:
I put WARNING message saying _!!WARNING!! tracer is on without DW routing method. Turn off history outputs for soluteFlux and soluteMass_ when a user turn on tracer, but not using DW, and make sure to turn off history outputs for soluteFlux and soluteMass